core/src/test/java/greencity/service/impl/UserServiceImplTest.java

```diff
@@ -737,4 +706,165 @@ public void getActivatedUsersAmountTest() {
         long activatedUsersAmount = userService.getActivatedUsersAmount();
         assertEquals(1L, activatedUsersAmount);
     }
+
+    @Test
+    void getProfilePicturePathByUserIdNotFoundExceptionTest() {
+        assertThrows(NotFoundException.class, () ->
+            userService.getProfilePicturePathByUserId(1L)
+        );
+    }
+
+    @Test
+    void getProfilePicturePathByUserIdTest() {
+        when(userRepo.getProfilePicturePathByUserId(1L)).thenReturn(Optional.of(anyString()));
+        userService.getProfilePicturePathByUserId(1L);
+        verify(userRepo).getProfilePicturePathByUserId(1L);
+    }
+
+    @Test
+    void updateUserProfilePictureTest() {
+        MultipartFile image = new MockMultipartFile("data", "filename.txt", "text/plain", "some xml".getBytes());
+        when(userRepo.findByEmail(anyString())).thenReturn(Optional.of(user));
+
+        try {
+            when(fileService.upload(image)).thenReturn(new URL("http://test.com"));
+        } catch (MalformedURLException e) {
```

Comment:
Do not catch exceptions in unit test, just add it to method signature.

Reply:
Fixed